ern-local-cli/src/lib/utils.js

```diff
@@ -635,6 +636,62 @@ async function performPkgNameConflictCheck (name: string) : Promise<boolean> {
   return true // If package name doesn't exist continue with command execution
 }
 
+function checkIfModuleSuffixExists (moduleName: string, moduleType: string): boolean {
+  if (moduleName) {
+    switch (moduleType) {
+      case ModuleTypes.MINIAPP:
+        return moduleName.toUpperCase().indexOf('APP') > -1
+      case ModuleTypes.API:
+        return moduleName.toUpperCase().indexOf('API') > -1
+      case ModuleTypes.JS_API_IMPL:
+        return moduleName.toUpperCase().indexOf('IMPL') > -1
+      case ModuleTypes.NATIVE_API_IMPL:
+        return moduleName.toUpperCase().indexOf('IMPL') > -1
+      default:
+        return false
+    }
+  }
+  return false
+}
+
+async function promptUserToUseSuffixModuleName (moduleName: string, moduleType: string): Promise<string> {
+  let message = ''
+  let suffixedModuleName = moduleName
+  if (moduleName) {
+    switch (moduleType) {
+      case ModuleTypes.MINIAPP:
+        suffixedModuleName = `${moduleName}App`
+        message = `We recommend suffixing the name of ${moduleName} with App, Do you want to use ${suffixedModuleName}?`
+        break
+      case ModuleTypes.API:
+        suffixedModuleName = `${moduleName}Api`
+        message = `We recommend suffixing the name of ${moduleName} with Api, Do you want to use ${suffixedModuleName}?`
+        break
+      case ModuleTypes.JS_API_IMPL:
+        suffixedModuleName = `${moduleName}Impl`
+        message = `We recommend suffixing the name of ${moduleName} with Impl, Do you want to use ${suffixedModuleName}?`
+        break
+      case ModuleTypes.NATIVE_API_IMPL:
+        suffixedModuleName = `${moduleName}Impl`
+        message = `We recommend suffixing the name of ${moduleName} with Impl, Do you want to use ${suffixedModuleName}?`
+        break
+      default:
+        // do nothing
```

Comment:
throw an error instead of doing nothing. if a non recognized moduleType is provided to this function, it should fails instead of silently doing nothing.